### PR TITLE
LPS-76721 Fix soy docs

### PR DESCRIPTION
**Pull request by pedroqueiroz**

### CODE REVIEW

**Comment by pedroqueiroz** [comment]
ci:test:sf

**Comment by pedroqueiroz** [comment]
ci:test:relevant

**Comment by pedroqueiroz** [comment]
ci:test:relevant

**Comment by pedroqueiroz** [comment]
ci:test:relevant

**Comment by pedroqueiroz** [comment]
Hey @vicnate5, are the tests good? Thx!

**Comment by pedroqueiroz** [comment]
Hi @clovisdasilvaneto , I'm closing this as we need to fix the issues Victor found. Thanks! :)

**Comment by clovisdasilvaneto** [comment]
No problem, I'll fix it 😎

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/templates/field-options-toolbar.soy [hunk 1]
```diff
@@ -45,5 +45,5 @@
 {template .field_settings_toolbar_item}
 	{@param option: [buttonClass: string, handler: string, label: string]}
 
-	<a class="dropdown-item {$option.buttonClass or ''}" data-handler="{$option.handler}" href="javascript:;" title="{$option.label}">{$option.label}</a>
+	<a class="dropdown-item form-ddm-settings-item {$option.buttonClass or ''}" data-handler="{$option.handler}" href="javascript:;" title="{$option.label}">{$option.label}</a>
```
Comment (pedroqueiroz) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/templates/field-options-toolbar.soy]: hey @clovisdasilvaneto , I think a better name for that would be: `ddm-form-field-actions-item`

**Pull request closed without merging**